Kickstarter-iOS/Views/Controllers/LoginToutViewController.swift

```diff
@@ -85,6 +88,11 @@ internal final class LoginToutViewController: UIViewController, MFMailComposeVie
       self.navigationItem.leftBarButtonItem = .close(self, selector: #selector(self.closeButtonPressed))
     }
 
+    _ = self.navigationController?.navigationBar
+      ?|> \.barTintColor .~ .clear
```

Comment:
I created a helper function `configureTransparentNavigationBar()` as an extension of `UINavigationBar` that you can use to set the navigation bar to be transparent. It's currently in the `feature-onboarding-personalization` branch, but maybe make a note here to update this once that's merged?

Reply:
Oh good to know! I wonder if I could cherry-pick the commit

Reply:
Actually I copied and pasted the extension code and this is what I got:

<img src="https://user-images.githubusercontent.com/3709676/77965934-1926b880-72b0-11ea-9637-e97f98f4dea7.png" width="50%">

Another solution would be to hide the navigation bar and add the buttons on the top corners. What do you think?

Reply:
@nino I think you need to remove the `baseControllerStyle` in order for it to work. `baseControllerStyle` is applying it's own navigation bar style that's overriding our setup.

Reply:
(Just FYI I don't think you meant to write my name. (I mean, I know _some_ Swift, but I really couldn't tell you if removing the `baseControllerStyle` would help))